config/vufind/permissions.ini

```
@@ -118,6 +123,13 @@ permission = access.StaffViewTab
 role[] = loggedin
 permission = feature.Favorites
 
+; By default, logged in users are allowed to use developer settings.
```

Should we expand this?

```suggestion
; By default, logged in users with verified email addresses are allowed to use developer settings.
```

---

...and should we maybe not say "By default" since if you comment this out, there will be no such restrictions? Maybe a better comment would be "This default example configuration allows logged in users with verified email addresses to use developer settings."